paper/paper.md

```diff
@@ -0,0 +1,338 @@
+ to gather their feedback.
+
+# Curriculum
+
+Our curriculum is tailored to people with no prior coding experience who want to
+learn how to use R programming for data analysis, visualization and the reporting of results (Figure 2).
+Not only do we aim to teach our learners the basics of performing empirical data analysis,
+we also seek to provide a rigorous framework for adhering to reproducible
+research principles that enable researchers to easily share their empirical work with others.
+
+![Curriculum overview. Created with BioRender.com.](curriculum-overview.png)
+
+## Learning Objectives
+
+The key learning objectives for our curriculum are:
+
+1. Create clear and informative data visualizations in R, starting with messy data.
+1. Perform version control using the Unix shell and git.
+1. Create reproducible reports using R Markdown.
+1. Share code with others on GitHub.
+
+We believe these skills provide learners with a solid foundation from which they can teach
+themselves any additional coding skills for future use.
+
+## Course Content
+
+Our curriculum consists of nine modules that cover software setup, data analysis and
+visualization, version control, sharing code, and writing reports (see below for more details). 
+We also set an overall goal for the workshop to make the content substantively interesting
+and relatable to a wide audience regardless of their original academic discipline or professional practice. Specifically, we task our learners with producing a fictitious report to the
+United Nations that examines the relationship between gross domestic product (GDP), life
+expectancy, and CO~2~ emissions.
+The R programming lessons take a "tidyverse first" approach
+[@robinson_teach_2017] to effectively and efficiently teach learners powerful tools for plotting and data analysis.
+The nine curriculum modules are:
+
+0. Setup
+1. Welcome
+1. R for plotting (uses the tidyverse R packages)
+1. The Unix shell
+1. Git and GitHub
+1. R for data analysis (uses the tidyverse R packages)
+1. Writing reports in R Markdown (uses the rmarkdown R package [@xie_r_2018])
+1. Group practice exercises
+1. Where to go from here
+
+Each lesson builds on the previous ones.
+The Unix shell, git, and GitHub are introduced using the files generated in the
+R for plotting lesson.
+The lesson content for subsequent modules is then intermittently committed and
+pushed to GitHub.
+The `Writing reports in R Markdown' lesson combines all of the skills learned previously to produce a
+report (as a pdf, Word, or html document) that one could share with the United Nations.
+Next, learners put everything they have learned into
+practice by forming small groups and working on practice problems that cover the entire course
+content.
+The workshop completes with a short module recapping everything that the curriculum covered as well as offering suggestions on how
+learners can continue to get help and keep learning once the workshop ends.
+
+## Instructional Design
```

Comment:
Right now the lists include both curriculum content and info about how we run the workshop. Think that's okay, or should we separate them out more?

Reply:
I think it would be less succinct if we separated them